packages/@lwc/ssr-compiler/src/compile-template/transformers/component/slotted-content.ts

```diff
@@ -94,9 +78,9 @@ export const Component: Transformer<IrComponent> = function Component(node, cxt)
                 draft.attributes = draft.attributes.filter((attr) => attr.name !== 'slot');
             });
             const slotContent = irToEs(clone, cxt);
-            return bAddContent(slotName, null, slotContent);
+            return b.expressionStatement(bAddContent(slotName, null, slotContent));
         } else {
-            return bAddContent(b.literal(''), null, irToEs(child, cxt));
+            return b.expressionStatement(bAddContent(b.literal(''), null, irToEs(child, cxt)));
```

Comment:
Wrapped these `CallExpression`s into `ExpressionStatement`s so the types could fit into a `Statement[]` that's passed to both `component.ts` and `lwc-component.ts`.

Reply:
I tried that but `esTemplate` still produces a `CallExpression`.